packages/ui-components/src/components/toolbar.tsx

```diff
@@ -376,7 +376,7 @@ export class ReactiveToolbar extends Toolbar<Widget> {
     this.popupOpener.hide();
     this._resizer = new Throttler(async (callTwice = false) => {
       await this._onResize(callTwice);
-    }, 300);
+    }, 500);
```

Comment:
Is this increase required?

Reply:
Not really, but as it looks pretty smooth with a longer delay, I thought it best to avoid excessive calculations.
I have no objection to revert it.

Reply:
That's fine then, if we get any user feedback on it we can adjust it